index.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML / CSS Quiz</title>
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<body>
+  <article class="product1"> 
```

Comment:
I guess at one time you were writing some css for `class=product1` `class=product2` & `class=product3` and forgot to remove them. You don't need those classes any more.